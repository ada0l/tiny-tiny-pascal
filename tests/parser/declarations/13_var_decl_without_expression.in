var a: Integer; b: Real;

begin end.
