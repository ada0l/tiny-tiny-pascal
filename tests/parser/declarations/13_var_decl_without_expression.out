main
   program name:
      without name
   decls:
      vars:
         var
            a 
            type:
               Integer 
            without exp
         var
            b 
            type:
               Real 
            without exp
   block:
      compounds statements